config/Dockerfiles/run_tests.sh

```diff
@@ -7,7 +7,7 @@ shift
 DISTROS=$*
 #TARGETS="dummy os-server-new os-sg-new os-vol-new"
 
-PROVIDERS="dummy openstack"
+PROVIDERS="dummy libvirt"
```

Comment:
Was this supposed to be a temporary change, or was having openstack there wrong to begin with?

Reply:
I believe it was a mistake to begin with.  The PROVIDERS in run_tests.sh don't affect what we run in jenkins, only when running locally with make test.  libvirt will work locally, the openstack provider here will only work if you have a openstack clouds.yaml in the keys directory.